cypress/integration/brewery/TableParameters-fullscreen.spec.js

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { Scenarios, ScenarioParameters, TableParameters, Login } from '../../commons/actions';
+import { BreweryParameters } from '../../commons/actions/brewery';
+import { stub } from '../../commons/services/stubbing';
+import { SOLUTIONS } from '../../fixtures/stubbing/TableParameters-fullscreen/solutions';
+
+const getFullscreenTable = TableParameters.getFullscreenTable;
+
+const getVipCustomersTable = () => {
+  return cy.get('[data-cy=table-vipCustomers]');
+};
+
+describe('check every TableToolbar component', () => {
+  before(() => {
+    stub.start({
+      GET_DATASETS: true,
+      GET_SCENARIOS: true,
+      GET_WORKSPACES: true,
+      GET_SOLUTIONS: true,
+      PERMISSIONS_MAPPING: true,
+    });
+    stub.setSolutions(SOLUTIONS);
+  });
+
+  beforeEach(() => {
+    Login.login();
+  });
+
+  after(() => {
+    stub.stop();
+  });
+
+  it('Edit a cell, and check if value as change in and out of fullscreen', () => {
+    Scenarios.getScenarioViewTab(60).should('be.visible');
+    ScenarioParameters.expandParametersAccordion();
+    BreweryParameters.switchToCustomersTab();
+    BreweryParameters.getCustomersTable().should('be.visible');
+    BreweryParameters.getCustomersTableLabel().should('be.visible').should('have.text', 'Customers');
+    BreweryParameters.getCustomersTableGrid().should('exist');
+    BreweryParameters.getCustomersFullscreenButton().should('not.exist');
+    BreweryParameters.importCustomersTableData('customers.csv');
+    TableParameters.importFile(getVipCustomersTable(), 'customers.csv');
```

Comment:
Why do we import the same file _customers.csv_ for the "vip customers" table? Its columns are different from the ones in the customers table, right?

Reply:
Now got the same, DONE